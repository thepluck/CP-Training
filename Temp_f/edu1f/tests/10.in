842 1
0 0
1 1
2 0
3 1
4 0
5 1
6 0
7 1
8 0
9 1
10 0
11 1
12 0
13 1
14 0
15 1
16 0
17 1
18 0
19 1
20 0
20 2
19 3
18 2
17 3
16 2
15 3
14 2
13 3
12 2
11 3
10 2
9 3
8 2
7 3
6 2
5 3
4 2
3 3
2 2
1 3
0 2
0 4
1 5
2 4
3 5
4 4
5 5
6 4
7 5
8 4
9 5
10 4
11 5
12 4
13 5
14 4
15 5
16 4
17 5
18 4
19 5
20 4
20 6
19 7
18 6
17 7
16 6
15 7
14 6
13 7
12 6
11 7
10 6
9 7
8 6
7 7
6 6
5 7
4 6
3 7
2 6
1 7
0 6
0 8
1 9
2 8
3 9
4 8
5 9
6 8
7 9
8 8
9 9
10 8
11 9
12 8
13 9
14 8
15 9
16 8
17 9
18 8
19 9
20 8
20 10
19 11
18 10
17 11
16 10
15 11
14 10
13 11
12 10
11 11
10 10
9 11
8 10
7 11
6 10
5 11
4 10
3 11
2 10
1 11
0 10
0 12
1 13
2 12
3 13
4 12
5 13
6 12
7 13
8 12
9 13
10 12
11 13
12 12
13 13
14 12
15 13
16 12
17 13
18 12
19 13
20 12
20 14
19 15
18 14
17 15
16 14
15 15
14 14
13 15
12 14
11 15
10 14
9 15
8 14
7 15
6 14
5 15
4 14
3 15
2 14
1 15
0 14
0 16
1 17
2 16
3 17
4 16
5 17
6 16
7 17
8 16
9 17
10 16
11 17
12 16
13 17
14 16
15 17
16 16
17 17
18 16
19 17
20 16
20 18
19 19
18 18
17 19
16 18
15 19
14 18
13 19
12 18
11 19
10 18
9 19
8 18
7 19
6 18
5 19
4 18
3 19
2 18
1 19
0 18
0 20
1 21
2 20
3 21
4 20
5 21
6 20
7 21
8 20
9 21
10 20
11 21
12 20
13 21
14 20
15 21
16 20
17 21
18 20
19 21
20 20
20 22
19 23
18 22
17 23
16 22
15 23
14 22
13 23
12 22
11 23
10 22
9 23
8 22
7 23
6 22
5 23
4 22
3 23
2 22
1 23
0 22
0 24
1 25
2 24
3 25
4 24
5 25
6 24
7 25
8 24
9 25
10 24
11 25
12 24
13 25
14 24
15 25
16 24
17 25
18 24
19 25
20 24
20 26
19 27
18 26
17 27
16 26
15 27
14 26
13 27
12 26
11 27
10 26
9 27
8 26
7 27
6 26
5 27
4 26
3 27
2 26
1 27
0 26
0 28
1 29
2 28
3 29
4 28
5 29
6 28
7 29
8 28
9 29
10 28
11 29
12 28
13 29
14 28
15 29
16 28
17 29
18 28
19 29
20 28
20 30
19 31
18 30
17 31
16 30
15 31
14 30
13 31
12 30
11 31
10 30
9 31
8 30
7 31
6 30
5 31
4 30
3 31
2 30
1 31
0 30
0 32
1 33
2 32
3 33
4 32
5 33
6 32
7 33
8 32
9 33
10 32
11 33
12 32
13 33
14 32
15 33
16 32
17 33
18 32
19 33
20 32
20 34
19 35
18 34
17 35
16 34
15 35
14 34
13 35
12 34
11 35
10 34
9 35
8 34
7 35
6 34
5 35
4 34
3 35
2 34
1 35
0 34
0 36
1 37
2 36
3 37
4 36
5 37
6 36
7 37
8 36
9 37
10 36
11 37
12 36
13 37
14 36
15 37
16 36
17 37
18 36
19 37
20 36
20 38
19 39
18 38
17 39
16 38
15 39
14 38
13 39
12 38
11 39
10 38
9 39
8 38
7 39
6 38
5 39
4 38
3 39
2 38
1 39
0 38
0 40
1 41
2 40
3 41
4 40
5 41
6 40
7 41
8 40
9 41
10 40
11 41
12 40
13 41
14 40
15 41
16 40
17 41
18 40
19 41
20 40
20 42
19 43
18 42
17 43
16 42
15 43
14 42
13 43
12 42
11 43
10 42
9 43
8 42
7 43
6 42
5 43
4 42
3 43
2 42
1 43
0 42
0 44
1 45
2 44
3 45
4 44
5 45
6 44
7 45
8 44
9 45
10 44
11 45
12 44
13 45
14 44
15 45
16 44
17 45
18 44
19 45
20 44
20 46
19 47
18 46
17 47
16 46
15 47
14 46
13 47
12 46
11 47
10 46
9 47
8 46
7 47
6 46
5 47
4 46
3 47
2 46
1 47
0 46
0 48
1 49
2 48
3 49
4 48
5 49
6 48
7 49
8 48
9 49
10 48
11 49
12 48
13 49
14 48
15 49
16 48
17 49
18 48
19 49
20 48
20 50
19 51
18 50
17 51
16 50
15 51
14 50
13 51
12 50
11 51
10 50
9 51
8 50
7 51
6 50
5 51
4 50
3 51
2 50
1 51
0 50
0 52
1 53
2 52
3 53
4 52
5 53
6 52
7 53
8 52
9 53
10 52
11 53
12 52
13 53
14 52
15 53
16 52
17 53
18 52
19 53
20 52
20 54
19 55
18 54
17 55
16 54
15 55
14 54
13 55
12 54
11 55
10 54
9 55
8 54
7 55
6 54
5 55
4 54
3 55
2 54
1 55
0 54
0 56
1 57
2 56
3 57
4 56
5 57
6 56
7 57
8 56
9 57
10 56
11 57
12 56
13 57
14 56
15 57
16 56
17 57
18 56
19 57
20 56
20 58
19 59
18 58
17 59
16 58
15 59
14 58
13 59
12 58
11 59
10 58
9 59
8 58
7 59
6 58
5 59
4 58
3 59
2 58
1 59
0 58
0 60
1 61
2 60
3 61
4 60
5 61
6 60
7 61
8 60
9 61
10 60
11 61
12 60
13 61
14 60
15 61
16 60
17 61
18 60
19 61
20 60
20 62
19 63
18 62
17 63
16 62
15 63
14 62
13 63
12 62
11 63
10 62
9 63
8 62
7 63
6 62
5 63
4 62
3 63
2 62
1 63
0 62
0 64
1 65
2 64
3 65
4 64
5 65
6 64
7 65
8 64
9 65
10 64
11 65
12 64
13 65
14 64
15 65
16 64
17 65
18 64
19 65
20 64
20 66
19 67
18 66
17 67
16 66
15 67
14 66
13 67
12 66
11 67
10 66
9 67
8 66
7 67
6 66
5 67
4 66
3 67
2 66
1 67
0 66
0 68
1 69
2 68
3 69
4 68
5 69
6 68
7 69
8 68
9 69
10 68
11 69
12 68
13 69
14 68
15 69
16 68
17 69
18 68
19 69
20 68
20 70
19 71
18 70
17 71
16 70
15 71
14 70
13 71
12 70
11 71
10 70
9 71
8 70
7 71
6 70
5 71
4 70
3 71
2 70
1 71
0 70
0 72
1 73
2 72
3 73
4 72
5 73
6 72
7 73
8 72
9 73
10 72
11 73
12 72
13 73
14 72
15 73
16 72
17 73
18 72
19 73
20 72
20 74
19 75
18 74
17 75
16 74
15 75
14 74
13 75
12 74
11 75
10 74
9 75
8 74
7 75
6 74
5 75
4 74
3 75
2 74
1 75
0 74
0 76
1 77
2 76
3 77
4 76
5 77
6 76
7 77
8 76
9 77
10 76
11 77
12 76
13 77
14 76
15 77
16 76
17 77
18 76
19 77
20 76
20 78
19 79
18 78
17 79
16 78
15 79
14 78
13 79
12 78
11 79
10 78
9 79
8 78
7 79
6 78
5 79
4 78
3 79
2 78
1 79
0 78
-1 78
-1 0
0 0 100 100