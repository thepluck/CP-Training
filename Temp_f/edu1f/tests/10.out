21.213203435596
